doc/md/versioned-migrations.mdx

```diff
@@ -211,6 +211,44 @@ h1:NRHsNIjvSSzprr/EzMdtszQg3t3pVLk4G4N1tX4rMfk=
 
 The full reference example exists in [GitHub repository](https://github.com/ent/ent/tree/master/examples/migration).
 
+### Verifying and linting migrations
+
+After generating our migration files with Atlas, we can run the [`atlas migrate lint`](https://atlasgo.io/versioned/lint)
+command that validate and analyze the contents of the migration directory and generate insights and diagnostics on the
+selected changes:
+
+1. Ensure the migration history can be replayed from any point at time.
+2. Protect from unexpected history changes when concurrent migrations are written to the migration directory by multiple
+team members. Read more about the consistency checks in the [section below](#atlas-migration-directory-integrity-file).
+3. Detect whether destructive or irreversible changes have been made or whether they are dependent on tables' contents
```

Comment:
Link to atlas 

https://atlasgo.io/lint/analyzers#destructive-changes

https://atlasgo.io/lint/analyzers#data-dependant-changes